### ISSUE TITLE
Sine, square, etc. wave generators (0xtob/#23)

### ISSUE THREAD

**Issue by asiekierka** [asiekierka/nitrotracker#47]
>how about some simple wave generations,. sin, square, saw, etc, a button under the  mic/record one would work,. it would open up a little synth interface,. something like Stomper or what ever you can do easily

**Comment by prayerie** [comment]
i know this is a 3 year old issue but i can work on this if it's still cool to add it

**Comment by asiekierka** [comment]
absolutely, but this might need some UX work. possibly a fullscreen popup box with a custom sample generation UI and preview?

**Comment by deadpahn** [comment]
Milkytracker has a waveform generator section if you are looking for inspo
